examples/react-native-example/README.md

```diff
@@ -5,7 +5,7 @@ React-native project running the Truvera wallet-sdk
 
 ```bash
 nvm use 20.2.0
```

Comment:
should we be updating node versions across the board?

Reply:
Good point, let me update this one and review the node versions for all of them